### ISSUE TITLE
[app-bar-profile-button] include visible text in accessible label

### ISSUE THREAD

**Issue by samrichardsontylertech** [tyler-technologies-oss/forge#974]
The accessible label of the profile button is currently the same as its tooltip — "View profile". Following WCAG 2.2 success criterion 2.5.3 the label should be or contain the visible text. While the usual content of the profile button (the user's initials) is arguably symbolic rather than "expressing something in human language" and therefore not subject to the criterion, it may still be beneficial to include it in the label. This would probably take the form of the accessible label for the button being changed to something like "View \<initials\>'s profile".
